scripts/apply-labels-3scale.sh

```diff
@@ -0,0 +1,165 @@
+            oc label imagestreamtag zync-database-postgresql:latest $THREESCALE_COMPONENT=system
+        else
+            echo Label $THREESCALE_COMPONENT=$COMPONENT added to secret $SECRET
+            oc label secret $SECRET $THREESCALE_COMPONENT=$COMPONENT
+        fi
+    done
+
+done
+
+# ConfigMap exceptions
+oc label configmap mysql-extra-conf $THREESCALE_COMPONENT=system
+oc label configmap mysql-main-conf $THREESCALE_COMPONENT=system
+oc label configmap redis-config $THREESCALE_COMPONENT=system
+
+# Route only one route has a label
+oc label route backend $THREESCALE_COMPONENT=backend
+
+
+################################################################################
+
+# Adding the threescale_component_element label
+
+for COMPONENT_ELEMENT in mysql redis production staging listener database cron worker app memcache sidekiq sphinx
+do 
+    for CONFIGMAP in $(oc get configmap | grep $COMPONENT_ELEMENT | awk '{print $1}')
+    do
+        echo Label $THREESCALE_COMPONENT_ELEMENT=$COMPONENT_ELEMENT added to configmap $CONFIGMAP 
+        oc label configmap $CONFIGMAP $THREESCALE_COMPONENT_ELEMENT=$COMPONENT_ELEMENT 
+    done
+    for POD in $(oc get po| grep $COMPONENT_ELEMENT | awk '{print $1}')
+    do
+        echo Label $THREESCALE_COMPONENT_ELEMENT=$COMPONENT_ELEMENT added to pod $POD 
+        oc label pod $POD $THREESCALE_COMPONENT_ELEMENT=$COMPONENT_ELEMENT 
+    done
+    for PVC in $(oc get pvc| grep $COMPONENT_ELEMENT | awk '{print $1}')
+    do
+        echo Label $THREESCALE_COMPONENT_ELEMENT=$COMPONENT_ELEMENT added to pvc $PVC 
+        oc label pvc $PVC $THREESCALE_COMPONENT_ELEMENT=$COMPONENT_ELEMENT 
+    done
+    for RC in $(oc get replicationController| grep $COMPONENT_ELEMENT | awk '{print $1}')
+    do
+        echo Label $THREESCALE_COMPONENT_ELEMENT=$COMPONENT_ELEMENT added to replicationController $RC 
+        oc label replicationController $RC $THREESCALE_COMPONENT_ELEMENT=$COMPONENT_ELEMENT 
+    done
+    for DC in $(oc get deploymentconfig | grep $COMPONENT_ELEMENT | awk '{print $1}')
+    do
+        echo Label $THREESCALE_COMPONENT_ELEMENT=$COMPONENT_ELEMENT added to deploymentconfig $DC 
+        oc label deploymentconfig $DC $THREESCALE_COMPONENT_ELEMENT=$COMPONENT_ELEMENT 
+    done
+    for ENDPOINTS in $(oc get endpoints | grep $COMPONENT_ELEMENT | awk '{print $1}')
+    do
+        echo Label $THREESCALE_COMPONENT_ELEMENT=$COMPONENT_ELEMENT added to endpoints $ENDPOINTS 
+        oc label endpoints $ENDPOINTS $THREESCALE_COMPONENT_ELEMENT=$COMPONENT_ELEMENT 
+    done
+    for SVC in $(oc get service | grep $COMPONENT_ELEMENT | awk '{print $1}')
+    do
+        echo Label $THREESCALE_COMPONENT_ELEMENT=$COMPONENT_ELEMENT added to service $SVC 
+        oc label service $SVC $THREESCALE_COMPONENT_ELEMENT=$COMPONENT_ELEMENT 
+    done
+done
+
+# Only one secret needs relabling 
+oc lable secret system-smtp $THREESCALE_COMPONENT_ELEMENT=smtp
```

Comment:
label is misspelled here and on all the lines below

Reply:
thanks first dry run show that up